### ISSUE TITLE
Error during `docker compose build ...`

### ISSUE THREAD

**Issue by totaltrash** [dunglas/symfony-docker#386]
Hey, thanks for symfony-docker, I'm looking forward to getting started.

I'm running through the getting started guide, and I'm getting an error at the second step (`docker compose build ...`):

`failed to solve: process "/bin/sh -c xcaddy build \t--with github.com/dunglas/mercure \t--with github.com/dunglas/mercure/caddy \t--with github.com/dunglas/vulcain \t--with github.com/dunglas/vulcain/caddy" did not complete successfully: exit code: 1`

Full log here: https://gist.github.com/totaltrash/6dc6665a8672cb00df1e00f8031c5553

Any ideas?

**Comment by totaltrash** [comment]
I'm gonna straighten out my docker installation... looks like it hasn't been updated in a while and guessing it's related

**Comment by totaltrash** [comment]
Yep, so updating docker composer from 2.10.2 to 2.15.1 hasn't actually helped. I've updated the gist with the most recent log (it doesn't look like much has changed though).

I'm a bit stumped. I've definitely got a functioning docker/compose installation... It feels like a network issue during caddy imports, maybe ipv6 related? I'll have a look at caddy doco to try and track it down, but if anyone here has seen anything like this, I'd love to hear.

**Comment by DocteurFlashbios** [comment]
@totaltrash you're not alone ;)
same thing here ...
`go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc: go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc@v1.4.0: verifying module: go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc@v1.4.0: Get "https://sum.golang.org/lookup/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc@v1.4.0": dial tcp [2a00:1450:4006:812::2011]:443: connect: cannot assign requested address`

**Comment by totaltrash** [comment]
Whew, thanks for confirming! Yeah, I'm seeing the same for the ip address I'm trying to connect to.

**Comment by totaltrash** [comment]
Bit more digging around:

```
PING sum.golang.org (142.250.70.209) 56(84) bytes of data.
64 bytes from mel05s01-in-f17.1e100.net (142.250.70.209): icmp_seq=1 ttl=60 time=18.1 ms
```

Total guess, but I wonder if the docker container is somehow getting stale DNS records (ie it's resolving sum.golang.org to some old data centres - yours in Ireland, and mine in Sydney)... When I get a moment I'm gonna install Go locally and run that xcaddy command and see what happens

**Comment by maxhelias** [comment]
It is probably a DNS problem and we cannot do anything herre. Sorry
